controls/overlay.rb

```diff
@@ -1083,7 +1083,7 @@
     sql = postgres_session(pg_dba, pg_dba_password, pg_host, input('pg_port'))
 
     describe sql.query('SHOW password_encryption;', [pg_db]) do
-      its('output') { should match /on|true/i }
+      its('output') { should match /on|true|scram-sha-256/i }
```

Comment:
Is there an updated STIG that has that listed as an option?  Cause if so, shouldn't we update the meta data here?  And what if someone's using a different, valid algorithm?  Is there a full collection listed somewhere that we can compare against instead of just this one?  If there isn't an official one, maybe we can turn it into a variable so that you can specify which algorithms are allowed?

Reply:
Not sure how much needs updated in the metadata for the overlay vs. baseline. This is the logic from the baseline.
https://github.com/mitre/crunchy-data-postgresql-stig-baseline/blob/main/controls/V-233596.rb

Reply:
Since the upstream has it similar to what you're doing here, I'm gonna make the review an 'approve' so that we're matching but leave the conversation unresolved so that you might make a new issue out of it as necessary based off of your research.